app/models/refile_error_search.rb

```diff
@@ -58,8 +58,8 @@ def per_page
 
   def get_refiles
     self.bearer     = self.assign_bearer
-    this_start = date_start.strftime('%Y-%m-%d') + 'T00:00:00-00:00'
-    this_end = date_end.strftime('%Y-%m-%d') + 'T23:59:59-00:00'
+    this_start = date_start.strftime('%Y-%m-%d') + 'T00:00:00-04:00'
```

Comment:
I got this error when trying to visit `/refile`, `no implicit conversion of Date into String`